beacon-chain/db/kv/backup.go

```diff
@@ -0,0 +1,40 @@
+package kv
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/boltdb/bolt"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+)
+
+const backupsDirectoryName = "backups"
+
+// Backup the database to the datadir backup directory.
+// Example for backup at slot 345: $DATADIR/backups/prysm_beacondb_at_slot_0000345.backup
+func (k *Store) Backup(ctx context.Context) error {
+	ctx, span := trace.StartSpan(ctx, "BeaconDB.Backup")
+	defer span.End()
+
+	backupsDir := path.Join(k.databasePath, backupsDirectoryName)
```

Comment:
Shouldnt backups live in another place other than the current data directory ?

Reply:
Addressed this feedback by changing `--clear-db` to only remove the beaconchain.db file.